src/makeCalculator.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+      return this;
```

Comment:
Add the blank line before the return, fix it everywhere
```suggestion
      this.result += value;
      
      return this;
```